.github/workflows/issue_title_check.yaml

```
@@ -0,0 +1,42 @@
+          echo "Title: $TITLE"
+          echo "Length: $LENGTH"
+
+          if [ "$LENGTH" -ge 60 ]; then
+            echo "too_long=true" >> "$GITHUB_OUTPUT"
+            exit 1
+          else
+            echo "too_long=false" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Post comment if title is too long
+        if: steps.title_check.outcomes == 'failure' # or "needs.title_check.result == 'failure'"
```

actionlint (1.7.4)

property "outcomes" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)